src/wallet/rpcwallet.cpp

```diff
@@ -3226,8 +3226,11 @@ UniValue signrawtransactionwithwallet(const JSONRPCRequest& request)
 
 static UniValue bumpfee(const JSONRPCRequest& request)
 {
-            RPCHelpMan{"bumpfee",
+    bool want_psbt = request.strMethod == "psbtbumpfee";
+
+            RPCHelpMan{request.strMethod,
```

Comment:
nit: fix pre-existing indentation (in separate commit)

Reply:
Done